src/app/views/apps-management/app-management/app-management.html

```diff
@@ -227,35 +227,46 @@ <h3>
           <div class='col-xs-12'>
             <mno-loader ng-show="vm.isOrderHistoryLoading" class='top-buffer-4'></mno-loader>
             <div ng-show="!vm.isOrderHistoryLoading" class='well order-histroy top-buffer-2'>
-              <div class='bottom-buffer-3'>
-                <div class='row subscription-event'>
-                  <div class='col-sm-4'>
-                    <label>
-                      {{ 'mno_enterprise.templates.dashboard.app_management.manage.tab.order_history.start_date' | translate }}
-                    </label>
-                  </div>
-                  <div class='col-sm-4'>
-                    <label>
-                      {{ 'mno_enterprise.templates.dashboard.app_management.manage.tab.order_history.offer_name' | translate }}
-                    </label>
-                  </div>
-                  <div class='col-sm-4'>
-                    <label>
-                      {{ 'mno_enterprise.templates.dashboard.app_management.manage.tab.order_history.status' | translate }}
-                    </label>
+              <div ng-if="!vm.subscriptionsHistory">
+                {{ 'mno_enterprise.templates.dashboard.app_management.manage.tab.order_history.no_history' | translate }}
+              </div>
+              <div ng-if="vm.subscriptionsHistory">
```

Comment:
Can we change to ```ng-hide="_.isEmpty(vm.subscriptionsHistory)"``` because I think it will instantiate to an empty array not false.

Reply:
Same reasoning as above.